minerl/herobraine/hero/handlers/agent/observations/inventory.py

```diff
@@ -79,11 +82,11 @@ def from_hero(self, obs):
         :param obs:
         :return:
         """
-        item_dict = self.space.no_op()
+        item_dict = defaultdict(str)
```

Comment:
```suggestion
        item_dict = {item_id: 0 for item_id in self.item_list}  # Much faster than Dict.no_op()
```

Should be just as fast, and more explicit for the user/developers.

Reply:
Good point - I've updated